tests/Sdk/Unit/Trace/SpanContextTest.php

```diff
@@ -31,18 +31,22 @@ public function testSpanContextCanCreateSampledSpans()
         $span = SpanContext::generate(true);
 
         $this->assertTrue($span->isSampled());
+
+        $span = SpanContext::generateSampled();
+
+        $this->assertTrue($span->isSampled());
     }
 
     /**
      * @test
      */
-    public function testDefaultSpansFromTracerAreNotSampled()
+    public function testDefaultSpansFromTracerAreSampled()
     {
         $tracer = $this->getTracer();
 
         $span = $tracer->startAndActivateSpan('test');
 
-        $this->assertFalse($span->isSampled());
+        $this->assertTrue($span->isSampled());
```

Comment:
I'm not sure about this. Actually this PR will change the default behavior(prior to this new spans were not sampled).
Is this a logic breaking change?

Reply:
It looks like the default sampler was declared to be `ParentOrElse(AlwaysOn)` - that was changed 3 days ago:

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#built-in-samplers

I think this behaviour is fine for now.  We can always change it later if need be.